packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1474,6 +1474,29 @@ class CompositedTransformFollower extends SingleChildRenderObjectWidget {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=T4Uehk3_wlY}
 ///
+///
+///{@tool sample --template=stateless_widget_scaffold_center.tmpl}
+///
+///In this example, the image is stretched to fill the entire [Container](https://master-api.flutter.dev/flutter/widgets/Container-class.html) which would not happen normally without using FittedBox.
```

Comment:
- Make sure that each line is shorter than 80 characters if possible
- The dartdocs automatically link widgets, so you do not need to include the URL like you would in a Github comment
```suggestion
/// In this example, the image is stretched to fill the entire [Container], which would 
/// not happen normally without using FittedBox.
```